.gitmodules

```diff
@@ -32,3 +32,6 @@
 	path = hcsshim
 	url = https://github.com/openshift/hcsshim
 	branch = master
+[submodule "cloud-provider-azure"]
+	path = cloud-provider-azure
+	url = https://github.com/kubernetes-sigs/cloud-provider-azure
```

Comment:
Should this be the openshift fork?

Reply:
Indeed, but we need to backport your changes. I'm working on this :)

Reply:
@Fedosin looks like you running into issues with the way `clonerefs` deals with merges that involves git sub-modules. Couple of suggestions:
- Try a local rebase and push to see if it helps
- Now that  openshift/cloud-provider-azure#23 has merged, try opening a new PR where you are just adding the `cloud-provider-azure` sub-module pointing at the OpenShift fork. This is just to check if the problem persists in a separate PR also.